### ISSUE TITLE
Support text-emphasis-position

### ISSUE THREAD

**Issue by yisibl** [postcss/autoprefixer#578]
Old Safari and Chrome support for -webkit-text-emphasis-position. These browsers support over and under as values, but not the added `left` and `right` values required by the spec.

input:
```css
em {
  text-emphasis-position: over right; /* Standard syntax in the specification */
}
```

output:

```css
em {
 -webkit-text-emphasis-position: over;
         text-emphasis-position: over right;
}
```

## Note

WebKit and Blink support over and under as values, but not the added left and right values required by the spec.

That is to say:

```css
-webkit-text-emphasis-position: under ≈ text-emphasis-position: under left;
-webkit-text-emphasis-position: over  ≈ text-emphasis-position: over right;
```

input:

```css
em {
  text-emphasis-position: under;
}
```

output:

```css
em {
 -webkit-text-emphasis-position: under;
         text-emphasis-position: under left;
}
```
[Demo](http://codepen.io/yisi/pen/GoKErz)

@zmmbreeze @ethantw Please review this.

[jquery.emphasis](https://github.com/zmmbreeze/jquery.emphasis/) and [han.css](https://github.com/ethantw/Han) muse be use standard grammar.

**Comment by ai** [comment]
Cool, we have Can I Use page for it: http://caniuse.com/#feat=text-emphasis

Try to fix it at weekend

**Comment by yisibl** [comment]
PR for caniuse: https://github.com/Fyrd/caniuse/pull/2123

**Comment by ai** [comment]
Done 5a9a8bc

**Comment by yisibl** [comment]
Are you going to support this?

input:

```css
em {
  text-emphasis-position: under;
}
```

output:

```css
em {
 -webkit-text-emphasis-position: under;
         text-emphasis-position: under left;
}
```

**Comment by ai** [comment]
We are not a polyfill or CSS linter. In prefix-less future CSS should be correct without Autoprefixer. So we should not fix unprefixed value.

**Comment by ai** [comment]
If it is very popular mistake, we can show warning for user. What do you think about it?

**Comment by yisibl** [comment]
OK, I know.

**Comment by yisibl** [comment]
> If it is very popular mistake, we can show warning for user. What do you think about it?

@ai  You can increase the warning, because WebKit only support one value.

**Comment by ai** [comment]
@yisibl but we specially generate `-webkit-text-emphasis-position: under;` for Safari. Why users will write `text-emphasis-position: under`? Idea of Autoprefixer to write CSS by W3C spec with `under left`.

**Comment by yisibl** [comment]
@ai Because WebKit and Blink only support the [old spec(2010)](http://www.w3.org/TR/2010/WD-css3-text-20101005/#text-emphasis-position) syntax.Many people do not know the new syntax, so there should be a warning.

For example [jquery.emphasis](http://zencode.in/jquery.emphasis/demo.html) and [han.css](https://github.com/ethantw/Han/blob/02b996115c713eee1291de5a27981cc7b0fb4a8e/src/sass/locale/_mixin.sass#L69-L71)  used the old syntax.

```sass
  @if ( $posi != null and $posi != none )
    -moz-text-emphasis-position:    $posi
    -webkit-text-emphasis-position: $posi
    text-emphasis-position:         $posi
```

**Comment by zmmbreeze** [comment]
:+1:

**Comment by ai** [comment]
Done d38bd4d

**Comment by ai** [comment]
Released in 6.2 version
